src/client/app/components/about.ts

```diff
@@ -13,15 +15,21 @@ import {Strings, getAvailableLanguages, getDisplayLanguage, setDisplayLanguage }
                     <div class="about__primary-text ms-font-xxl">{{config?.build?.name}}</div>
                     <div class="profile__tertiary-text ms-font-m">{{strings.userId}}: ${storage.user}</div>
                     <div class="about__secondary-text ms-font-l">Version: {{config?.build?.version}}
-                    <br/><span class="ms-font-m">(Deployed {{config?.build?.humanReadableTimestamp}})</span>
-                    <br/><span class="ms-font-m">{{config?.editorUrl}}</span>
+                        <br/><span class="ms-font-m">(Deployed {{config?.build?.humanReadableTimestamp}})</span>
                     </div>
                     <pre class="about__tertiary-text ms-font-m">{{cache}}</pre>
                     <div class="about__language">
                         <select class="about__language-select ms-font-m" [(ngModel)]="currentChosenLanguage" (change)="setLanguage($event.target.value)">
                             <option *ngFor="let l of availableLanguages" [value]="l.value">{{l.name}}</option>
                         </select>
                     </div>
+                    <div class="about__environment">
+                        <span class="ms-font-m about__environment-text">{{config?.editorUrl}}</span>
+                        <br/><label class="ms-font-m about__environment-text">${Strings().aboutSwitchEnvironment}</label>
```

Comment:
I don't think this (`${Strings().aboutSwitchEnvironment}`) will work.  In fact, I'm kind of suprised it didn't fail on you at runtime.  You should use it like this instead:  `{{strings.aboutSwitchEnvironment}}`